python/cudf/cudf/tests/test_concat.py

```diff
@@ -300,7 +300,9 @@ def test_pandas_concat_compatibility_axis1_eq_index():
     s2 = gd.Series(["a", "b", "c"], index=[1, 1, 1])
     ps1 = s1.to_pandas()
     ps2 = s2.to_pandas()
-
+    expect = None
+    got = None
```

Comment:
Lets get rid of these and just remove the assertion instead and this is good to go

Reply:
Ok.